src/pages/CoursePage/CoursePage.scss

```diff
@@ -0,0 +1,152 @@
+.course__header {
+  width: 100vw;
+  color: #fff;
+  width: 100vw;
+  background-color: #29303b;
```

Comment:
You can use a theme with colors.
In the future, if your design will be changed, it would be easier to change color.